azure/src/integration/java/org/apache/iceberg/azure/adlsv2/AzuriteContainer.java

```diff
@@ -33,7 +33,7 @@ public class AzuriteContainer extends GenericContainer<AzuriteContainer> {
 
   private static final int DEFAULT_PORT = 10000; // default blob service port
   private static final String DEFAULT_IMAGE = "mcr.microsoft.com/azure-storage/azurite";
-  private static final String DEFAULT_TAG = "3.34.0";
+  private static final String DEFAULT_TAG = "3.35.0";
```

Comment:
can you also please bump the version as was done by https://github.com/apache/iceberg/pull/13724